include/z64save.h

```diff
@@ -186,6 +186,15 @@ typedef enum {
     /* 0xFF */ BTN_DISABLED = 0xFF
 } ButtonStatus;
 
+typedef enum {
+    /* 0x00 */ CHAMBER_CS_FOREST,
+    /* 0x01 */ CHAMBER_CS_FIRE,
+    /* 0x02 */ CHAMBER_CS_WATER,
+    /* 0x03 */ CHAMBER_CS_SPIRIT,
+    /* 0x04 */ CHAMBER_CS_SHADOW,
+    /* 0x05 */ CHAMBER_CS_LIGHT
+} ChamberCutsceneNum;
```

Comment:
I would add a comment near `chamberCutsceneNum` in gSaveContext mentioning this enum

Reply:
The enum name being the same as the variable should cover this.

Reply:
I don't know about you but I don't `Ctrl+Shift+F` every struct member I come across to find out if they have an enum somewhere :p (plus that would find all the usage of `chamberCutsceneNum` in code)

Reply:
context will make this very clear, youd be able to locate the enum by looking at an existing usage of it. If we start pointing out the corresponding enum for everything in the codebase it would start to get messy fast, imo